### PR TITLE
chore: Use new input & output assets endpoints for tasks

### PR DESCRIPTION
**Pull request by Milouu**
Signed-off-by: Milouu <milan.roustan@owkin.com>

## Description

Use new endpoints to get assets for task inputs & outputs

## Notes for developers and reviewers:

-   Think to update CHANGELOG.md before merge if needed !

### CODE REVIEW

**Review by jmorel — CHANGES_REQUESTED** [review]

File: src/routes/tasks/components/TaskInputsDrawerSection.tsx [hunk 1]
```diff
@@ -251,26 +308,29 @@ const TaskInputSectionEntry = ({
...
+    algoInput: AlgoInputT;
     inputs: TaskInputT[];
+    inputsAssets: TaskIOT[];
 }): JSX.Element => {
     const icon = inputIcon(algoInput.kind);
     const title = identifier;
 
     if (algoInput.multiple || inputs.length > 1) {
-        //  '|| inputs.length > 1' is for resilience to inconsistent data
+        //  '|| Object.entries(inputs).length > 1' is for resilience to inconsistent data
```
Comment (jmorel) [inline src/routes/tasks/components/TaskInputsDrawerSection.tsx:322]: Is this change in the comment still relevant?

Comment (jmorel) [reply src/routes/tasks/components/TaskInputsDrawerSection.tsx:322]: Or maybe the check should be on Object.keys(input)

**Review by jmorel — COMMENTED** [review]
One small last comment, otherwise it's looking good !

File: src/routes/tasks/components/TaskInputsDrawerSection.tsx (hunk 1 above)
Comment (Milouu) [reply src/routes/tasks/components/TaskInputsDrawerSection.tsx:322]: No change in the comment is no longer relevant, nice catch. Left over from when i tried to pass inputs as an object.

**Review by jmorel — APPROVED** [review]

**Pull request merged**